build/import/Packages.targets

```diff
@@ -60,23 +61,23 @@
     <PackageReference Update="Microsoft.VisualStudio.Shell.Framework"                                 Version="17.1.0-preview-2-32002-062" />
     <PackageReference Update="Microsoft.VisualStudio.Telemetry"                                       Version="16.4.13" />
     <PackageReference Update="Microsoft.VisualStudio.TemplateWizardInterface"                         Version="17.1.0-preview-2-32002-062" />
-    <PackageReference Update="Microsoft.VisualStudio.Threading"                                       Version="17.1.42-alpha" />
-    <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers"                             Version="17.1.42-alpha" />
+    <PackageReference Update="Microsoft.VisualStudio.Threading"                                       Version="17.2.10-alpha" />
+    <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers"                             Version="17.2.10-alpha" />
     <PackageReference Update="Microsoft.VisualStudio.Utilities"                                       Version="17.1.0-preview-2-32002-062" />
-    <PackageReference Update="Microsoft.VisualStudio.Validation"                                      Version="17.0.42" />
+    <PackageReference Update="Microsoft.VisualStudio.Validation"                                      Version="17.0.46" />
     <PackageReference Update="Microsoft.VisualStudio.XmlEditor"                                       Version="17.1.0-preview-2-32002-062" />
     <PackageReference Update="System.Threading.Tasks.Dataflow"                                        Version="6.0.0" />
     <PackageReference Update="Microsoft.VSDesigner"                                                   Version="17.0.0-preview-2-31223-026" />
     <PackageReference Update="VsWebSite.Interop"                                                      Version="16.8.30523.219"/>
 
     <!-- https://github.com/dotnet/roslyn/issues/53877 -->
-    <PackageReference Include="StreamJsonRpc"                                                         Version="2.8.28" />
+    <PackageReference Include="StreamJsonRpc"                                                         Version="2.10.42" />
 
     <!-- CPS -->
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem"                                   Version="17.1.221-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Analyzers"                         Version="17.1.221-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Query"                             Version="17.1.221-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.SDK.Tools"                         Version="17.1.221-pre" />
+    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem"                                   Version="17.2.192-pre-gd98aebaca6" />
```

Comment:
This CPS package version are for testing. At the end I will put the latest CPS versions.